blocks/cards/cards.js

```diff
@@ -2,6 +2,7 @@ import { createOptimizedPicture } from '../../scripts/aem.js';
 import { moveInstrumentation } from '../../scripts/scripts.js';
 
 export default function decorate(block) {
+  if (block.classList.contains('hero') && block.classList.contains('cards')) block.classList.remove('cards');
```

Comment:
This is confusing. Why are we doing it this way?

Reply:
Removed it. Earlier, i added to apply styles based on the view selected in author. Since, I don't want to have cards CSS to be applied.